heartbeat/VirtualDomain

```diff
@@ -389,20 +422,36 @@ set_util_attr() {
 	fi
 }
 
+unset_util_attr() {
+	local attr=$1
+	local cval outp
+
+	outp=$(crm_resource --resource=$OCF_RESOURCE_INSTANCE --utilization --delete-parameter=$attr 2>&1) ||
+	ocf_log warn "crm_resource failed to unset utilization attribute $attr: $outp"
+}
+
 update_utilization() {
 	local dom_cpu dom_mem
 
 	if ocf_is_true "$OCF_RESKEY_autoset_utilization_cpu"; then
 	   dom_cpu=$(LANG=C virsh $VIRSH_OPTIONS dominfo ${DOMAIN_NAME} 2>/dev/null | awk '/CPU\(s\)/{print $2}')
```

Comment:
Can you update these sections? They seem to combine tabs with spaces.

Reply:
You are correct.  I missed that when I replaced the other spaces.  New commit coming.